### PR TITLE
Added awscli2 package test suite.

### PR DESCRIPTION
**Pull request by narmaku**
Related task: CLOUDX-927

### CODE REVIEW

File: test_suite/package/test_awscli2.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+from lib import test_lib
+
+
+@pytest.mark.package
+@pytest.mark.run_on(['>=rhel9.5'])
+class TestsAwsCli2:
+    @pytest.fixture(scope='module', autouse=True)
+    def import_aws_credentials(self, host):
+        # Generate temporary credentials for this test.
+        civ_local_command_to_run = 'aws sts get-session-token --duration-seconds 3600 --output json'
```
Comment (narmaku) [inline test_suite/package/test_awscli2.py:14]: Tests take less than 1 minute to run, so maybe we should reduce this number to something much smaller? E.g. `90`

**Comment by ConorC117** [comment]
Code looks great to me! I had a good think about other worthwhile test cases to include, but to the best of my knowledge, I think the current tests are enough to ensure the package is working properly :)

File: test_suite/package/test_awscli2.py (hunk 1 above)
Comment (ConorC117) [reply test_suite/package/test_awscli2.py:14]: I think reducing the number to something much smaller is a good idea too. I'd probably set it to 90 like you suggested or maybe 120 to be extra cautious!

Comment (narmaku) [reply test_suite/package/test_awscli2.py:14]:
```suggestion
        civ_local_command_to_run = 'aws sts get-session-token --duration-seconds 120 --output json'
```

**Pull request merged**